### PR TITLE
Merge ApplicationPolicy into EnrollmentPolicy

### PR DESCRIPTION
**Pull request by TThurston**
Removes final reference to front-office-core

### CODE REVIEW

File: app/policies/digital_services_core/enrollment_policy.rb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 module DigitalServicesCore
-  class EnrollmentPolicy < FrontOfficeCore::ApplicationPolicy
+  class EnrollmentPolicy
+    CRUD_ACTIONS = %i[new create edit update destroy show index].freeze
```
Comment (lewispb) [inline app/policies/digital_services_core/enrollment_policy.rb]: I wonder what this is for? Doesn't seem to be used.

**Comment by lewispb** [comment]
Is it possible to upgrade Rails to 4.2.7.1?

File: app/policies/digital_services_core/enrollment_policy.rb (hunk 1 above)
Comment (TThurston) [inline app/policies/digital_services_core/enrollment_policy.rb]: Coming in a separate branch / PR :)

**Comment by lewispb** [comment]
:ship:  it, but needs a rebase

**Pull request merged**